script/import_forecasts.rb

```diff
@@ -0,0 +1,62 @@
+require "optparse"
+require_relative "../config/environment"
+
+parser = OptionParser.new { |args|
+  args.on "-f", "--fund FUND"
+  args.on "-o", "--organisation ORGANISATION"
+  args.on "-q", "--quarter QUARTER", Integer
+  args.on "-y", "--year YEAR", Integer
+  args.on "-i", "--input FILE"
+}
+
+options = {}
+parser.parse!(into: options)
+
+fund = Activity.fund.by_roda_identifier(options.fetch(:fund))
+organisation = Organisation.find_by(name: options.fetch(:organisation))
```

Comment:
I like fetching by name, rather than ID. Even though we have to find the exact name, it seems clearer

Reply:
My thinking here is I want the person running this to be really sure about which report they're adding the data to, and so I did this by having them type in the names of the fund and organisation, and then confirming that the selected report matches what they were intending. I'm sure there's lot of other UI designs one could do here and I want to make this as safe as possible, so happy to add/change anything that would help with that.